### PR TITLE
updated undead eye

### PR DESCRIPTION
**Pull request by GildedToxin**
UNDEAD EYE DONE!!!

### CODE REVIEW

**Comment by GildedToxin** [comment]
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

**Review by Jonah-Hansen — APPROVED** [review]

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 1]
```diff
@@ -862,9 +989,66 @@
...
 				"5DA4FF9594050625"
 			]
+			description: [
+				"&4Requires Nether Access&r"
+				""
+				"• Soul Bead - Dropped by &cWraiths&r"
+				""
+				"• Skreecher Soul - Dropped by &cSkreechers&r"
+				""
+				"• Cerberous Fang - Dropped by &4Cerberous&r"
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:999]: it should be "Cerberus" not "Cerberous" ?

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 2]
```diff
@@ -709,18 +709,19 @@
...
-				"• Ectoplasm x4:Drops from spirits in the soulsand valley."
-				"• Spirit eye: Craft 4 salt clumps dropped from soul sand valley Embodies, creatures made of soul sand."
-				"• Ancient Fruit:Found on ancient trees, the saplings of which can be found in Ancient Cities. Gives xp when eaten. Use the Nature's Compass to find a deep dark, and keep trying different deep darks until you find one. Most A. cities spawn under mountains."
-				"• Soul Bead:dropped from Wraiths in the soul sand valleys. Will fly toward the nearest nether fortress when used."
-				"• Skreecher soul: Drops from skreechers, creatures in the deep dark ceiling that shriek to summon the warden upon sighting you. Take them out with a bow before they see you. Use the Nature's Compass to find a deep dark, and keep trying different deep darks until you find one. Most A. cities spawn under mountains."
-				"* Cerebus Fang: Wave 5 of the undead army, summonable with cloth that drops from night mobs or naturally spawned by killing a lot of undead at night sometimes. Notifies you in chat when they're approaching."
+				"&4Requires Nether Access&r"
+				""
+				"• Frozen Rotten Flesh - Dropped by &cForstbitten&r"
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:714]: should be "Frostbitten" not "Forstbitten" ?

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 3]
```diff
@@ -810,14 +861,58 @@
...
 			}]
-			title: "Used in Undead Eye"
+			title: "Necronium Ingot (Undead Eye)"
 			x: 12.0d
 			y: -2.5d
 		}
 		{
 			dependencies: ["5DA4FF9594050625"]
+			description: [
+				"This item can be obtainted by killing &cForstbitten&r, which can be found in cold and snowy biomes at night"
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:871]: Should be "Frostbitten" not "Forstbitten" ?

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 4]
```diff
@@ -935,14 +1217,46 @@
...
 			}]
-			title: "Used in Undead Eye"
+			title: "Soul Bead (Undead Eye)"
 			x: 12.25d
 			y: 2.75d
 		}
 		{
 			dependencies: ["5DA4FF9594050625"]
+			description: [
+				"This item can be obtainted by cutting down Ancient Trees - this log extinct species of tree can be revitalized from &eAncient Saplings&r found in Ancient Cities. "
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:1227]: It should be "long extinct species" not "log extinct species" ?

Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:1227]: also "obtained" from "obtainted" ?

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 5]
```diff
@@ -973,14 +1316,30 @@
...
+				item: {
+					Count: 1b
+					ForgeCaps: {
+						"dungeons_libraries:built_in_enchantments": { }
+					}
+					id: "majruszsdifficulty:undead_army_treasure_bag"
+				}
+				type: "item"
+			}]
+			subtitle: "Droppd by Cerberus"
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:1342]: "Dropped by Cerberus" from "Droppd by Cerberus" ?

**Review by RUsami — COMMENTED** [review]
found some spelling error I think ?

**Pull request merged**